### DISCUSSION TITLE
Uncaught Exception -- Defined Twice

### DISCUSSION THREAD

**Discussion by cherryglazer** [Monika-After-Story/MonikaModDev#9662] [QA - tech support + general] [answered]
Hello!

I'm having an issue with opening MAS. When I open it, this message pops up [attached]:
[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/9750037/traceback.txt)

Does anyone know what could be wrong?

Thank you in advance!

**Comment by Booplicate** [comment]
Seems like you installed MAS incorrectly, try to use the installer.

**Comment by cherryglazer** [comment]
This was after I used the installer :/

**Comment by Booplicate** [comment]
When you selected a directory, did it tell you the directory is invalid?

**Comment by cherryglazer** [comment]
It did. I'm using the installer again, but this time to my main directory (the invalid message didn't pop up).

**Comment by cherryglazer** [comment]
It installed correctly, but another error message popped up:
[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/9750177/traceback.txt)

**Comment by Booplicate** [comment]
Can you show your base directory and `game/` folder?

**Comment by cherryglazer** [comment]
Sure thing!
![image](https://user-images.githubusercontent.com/110358748/194990510-45357759-b912-4cec-ae42-dfbc033cd983.png)
![image](https://user-images.githubusercontent.com/110358748/194990582-3850b017-3f33-40e4-a450-7a34fc5a5135.png)

**Comment by Booplicate** [comment, marked as answer]
At some point in the past you've installed the source version, which is why it fails to launch now (aside from possible save data issues you could get). Even if you reinstall using the installer, the issues from the source version won't go away.
Make a fresh DDLC install, then install MAS using the installer.

**Comment by cherryglazer** [comment]
I created a new install & moved my persistents over, but this pop-up showed:
![image](https://user-images.githubusercontent.com/110358748/195730319-789e64ea-c673-4195-b925-7c9073bb3cc6.png)

**Reply by Booplicate** [reply]
Your install looks incorrect, files/folders all over the place.

**Comment by cherryglazer** [comment]
Hmm, I just moved over my old submods folder into "game" & moved the non-folder material into "extra". Should I take them out?

**Reply by Booplicate** [reply]
I don't know what you moved, but that's not part of MAS and is causing issues. Maybe you installed it incorrectly, or maybe it has some bugs. Make a clean install again. If you need support for those external additions, you'll have to go to the creator of it.

**Comment by cherryglazer** [comment]
It worked -- thank you so much for your assistance !!! I just needed to reinstall my submods :)
